draft-btw-add-home.txt

```diff
@@ -988,26 +990,24 @@ Internet-Draft       Encrypted DNS in Home Networks        November 2020
 
    o  Locator: 2001:db8:1::1
 
-   The client requests "https://doh.example.com/.well-known/resinfo" to
-   retrieve the list of the URI Templates associated with this
-   discovered server.
+   The client will use DEER [I-D.pauly-add-deer] to discover the DoH
+   templates supported by the DNS server at the Locator address.  In
+   addition to the checks included in DEER, clients should verify the
+   ADN matches the hostname in the DEER-provided templates.  The IP
+   address of the DEER-discovered resolver may differ from the Locator
+   field value so long as the domain name matches the ADN.  This will
+   allow ISPs to have local forwarders send clients to off-network but
```

Comment:
Not just off-network, there could be all kinds of reasons why the IP address is different.

Reply:
I have generalized this text.